README.md

```diff
@@ -0,0 +1,194 @@
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Author](#author)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+
+# 📖 [Blog-app-Rails] <a name="about-project"></a>
+
+> This is a blog app built with Ruby on Rails. It allows users to create an account, create posts, and comment on other posts.
+
+## 🛠 Built With <a name="built-with"></a>
+
++ Ruby
++ Ruby on Rails
++ PostgreSQL
++ Rspec (for testing)
+
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- Users can create an account
+- Users can create posts
+- Users can comment on posts
+- Users can like posts
+- Users can follow other users
+- Users can see posts from users they follow
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> [Live Demo Link](https://blog-app-rails.herokuapp.com/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running follow these simple example steps.
+
+### Setup <a name="setup"></a>
+
+- Clone the repository with `git clone
+
+### Prerequisites <a name="prerequisites"></a>
+
+- Ruby: 2.7.0
+- Rails:
+- PostgreSQL
+- Rspec
+
+### Install <a name="install"></a>
+
+- Install gems with:
+
+```
+bundle install
+```
+
+- Setup database with:
+
+```
+   rails db:create
+   rails db:migrate
+```
+
+### Usage <a name="usage"></a>
+
+- Start server with:
+
+```
+   rails server
+```
+
+- Open `http://localhost:3000/` in your browser.
+
+### Run tests <a name="run-tests"></a>
+
+- To run tests, run:
+
+```
+   rpsec --format documentation
+```
+
+### Deployment <a name="triangular_flag_on_post-deployment"></a>
+
+- Deployed to Render.com
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Author <a name="author"></a>
+
+👤 **ANDY MENUTTI**
+
+- GitHub: [@andym80](https://github.com/andym80)
+- LinkedIn: [Andres Menutti](https://linkedin.com/in/andres-menutti)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- Add a search bar
+- Add a profile page
+- Add a notifications page
+- Add a share feature
+- Add a follow feature
+- Add a sign up feature
+- Add a sign in feature
+- Add a sign out feature
+- Add a sign in with Discord feature
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
```

Comment:
- [ ] Kindly fix your repository issues page link because it is broken :fearful: Kindly fix it to make your readme descriptive :blush: